include/datafile/UpgradeTo13Misc.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * UpgradeTo13Misc.h
+ *   Functor for upgrading data files for BLURB
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#pragma once
+
+#ifndef LMMS_CLASS_NAME_GUARD_H
+#define LMMS_CLASS_NAME_GUARD_H
```

Comment:
One of the reasons why the template file is not a great idea IMO, lol.
```suggestion
#ifndef LMMS_UPGRADE_13_MISC_H
#define LMMS_UPGRADE_13_MISC_H
```